script/setup.sh

```diff
@@ -65,6 +65,11 @@ case "$(uname -s)" in
 	# xpath for xml parsing
 	URL_NASR_CURRENT=$(curl -s $URL_NASR_EDITION | xpath -q -e 'string(//productSet/edition/product/@url)')
      ;;
+
+   MINGW*)
+	# MINGW (Windows) does not have xpath installed by default, so use grep and sed. This is somewhat hacky.
+	URL_NASR_CURRENT=$(curl -s $URL_NASR_EDITION | grep -o 'url=".\{0,256\}\.zip' | sed -e 's/^url="//')
```

Comment:
Thoughts on requiring xpath to be installed for a MINGW install?

Reply:
How is xpath usually installed within Linux? Through npm?

Reply:
Ok, I figured out how to install xpath for MINGW. The solution I found was to download [Strawberry Perl](https://strawberryperl.com/) as a standalone program within Windows, then use CPAN (provided by perl) to install xmlpath. Requiring MINGW users to install xpath would prevent the hacky code workaround, so this would probably be best.

Reply:
While I'm supportive of strong use of `grep` and `sed`, I'm concerned about how "hacky" the implementation is here. When you say "somewhat hacky," can you please elaborate on what that means? I'm trying to gauge if adding a StrawBerry Perl is a requirement (or assumed install) for MINGW users.

Reply:
Sure. What I mean by this is that the code relies upon the assumption that the layout of the XML in the FAA NASR page will not change. The script grabs the first url in the page and returns it, which could be problematic if the XML were modified at a later date, say to add an additional product url. In contrast, xpath explicitly follows the XML tree to find the needed url, making the xpath implementation unaffected to future changes.

Reply:
Would you consider requiring Strawberry Perl to be onerous or burdensome as a MINGW user?

Reply:
No. I think it would be a reasonable requirement. It takes only a few minutes to install it and xpath.

Reply:
Okay, let's do that then. Please update this script to use `xpath`. Also in the `README`, please update the Initial Setup section for MINGW users. The README should note the Strawberry Perl requirement.